Source/WebKit/UIProcess/API/glib/WebKitWebsiteDataManager.cpp

```diff
@@ -506,6 +552,12 @@ WebKit::WebsiteDataStore& webkitWebsiteDataManagerGetDataStore(WebKitWebsiteData
         if (priv->domCacheDirectory)
             configuration->setCacheStorageDirectory(FileSystem::stringFromFileSystemRepresentation(priv->domCacheDirectory.get()));
 #endif
+        if (priv->originStorageRatio >= 0.0 && priv->originStorageRatio <= 1.0)
+            configuration->setOriginQuotaRatio(priv->originStorageRatio);
```

Comment:
With the current values this is always true, since 0 is the min and 1 the maximum. If we change to use -1, here we would only need to check if != -1

Reply:
Maybe better would be check if >=0? So everything <0 will be ignored